models/mrsegmentator/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM mhubai/base:latest
+
+# Update Python, nnUNet needs Python >= 3.9
+#RUN apt-get update && apt-get install -y \
+#    python3.11 \
+#    python3-pip
+    
+# clone mhub implementation
+ARG MHUB_MODELS_REPO
+RUN buildutils/import_mhub_model.sh mrsegmentator ${MHUB_MODELS_REPO}
+
+# Install MRSegmentator
+RUN uv pip install -p /app/.venv mrsegmentator
+
+# Execute mrsegmentator once to download the weights, this will fail to run but download regardless
+RUN touch .temp_image.nii.gz
+RUN mrsegmentator -i .temp_image.nii.gz; exit 0
+RUN rm .temp_image.nii.gz
```

Comment:
Could this be the issue? I am not sure if mrsegmentator is available, try `uv run mrsegmentator` if it's installed in the `.venv`.

Reply:
I don't think this would be the issue, these lines seem to run fine when building the container, and there is no error message produced. The problem mainly is that the dicomsort module does not seem to find any data. I will still revise and use uv run mrsegmentator.